testsuite/MDAnalysisTests/analysis/test_diffusionmap.py

```diff
@@ -80,3 +80,15 @@ def test_transform(u, dmap):
     dmap.run()
     diffusion_space = dmap.transform(n_eigenvectors, 1)
     assert diffusion_space.shape == (eigvects.shape[0], n_eigenvectors)
+
+def test_long_traj(u):
+    with pytest.warns(UserWarning, match='The distance matrix is very large'):
+        dmap = diffusionmap.DiffusionMap(u)
+        dmap._dist_matrix.run()
```

Comment:
So it's not exactly a big thing (it only makes execution go down from 555 ms of CPU time to 27.7 ms), but since we only really care about the user warning, it might make sense to just put a `stop=1` argument on that `run` call.

Reply:
Thanks, that's a good point. Will keep it in mind.